assets/javascripts/modules/ajaxFormSubmit.js

```diff
@@ -128,15 +135,21 @@ var ajaxFormSubmit = {
   },
 
   serializeForAjax: function(formScope) {
-    var ret = ['isajax=true'];
+    var result = ['isajax=true'];
```

Comment:
Not really but in one of previous PRs it was mentioned to user more sensible name